src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -198,7 +198,24 @@
 			Canvas.ZIndex="0"
 			EmptyTextType="{x:Bind ParentShellPageInstance.FilesystemViewModel.EmptyTextType, Mode=OneWay}" />
 
-		<TeachingTip
+        <Border>
+            <StackPanel
```

Comment:
Why was this moved out of the footer?

Reply:
Because when setting it inside the footer the alignment was depending on the scroll view and couldn't center it depending on the window size